app/build.gradle

```diff
@@ -37,6 +37,12 @@ android {
         checkReleaseBuilds false
         abortOnError false
     }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
```

Comment:
iirc the tests are only checking if the arithmetic is working or not. Isn't it a lot of overhead on our build times to bring in all the androidResoruces ?

Reply:
we need it or else we cannot build the project. We are using [Roboelectric](http://robolectric.org/) as our unit test framework. It does not work with Android Asset Packaging Tool 2.0 (AAPT2). This causes a `Resources$NotFoundException`. A fix is to play with the env variables or include the resources in the gradle. I picked the latter for simplicity.

Sources: 
1. https://stackoverflow.com/questions/45922916/robolectric-resourcesnotfoundexception-string-resource-id-with-android-gradle
2. https://issuetracker.google.com/issues/64107525#comment3
3. https://android.jlelse.eu/make-robolectric-compatible-with-latest-gradle-tools-3-0-0-eca085681b3b

Reply:
Thanks kindly for clarifying this. Appreciate the sources hoorah